public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json

```diff
@@ -122,6 +122,26 @@
       },
       "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
     },
+    {
+      "path": "resourcepickerdata",
```

Comment:
I think we should call this azureresourcegraph

Reply:
What if we add support for multi-verb routes? For example:
```
"path": "azure-management-api",
"method": "GET,POST"
```

Reply:
That would be nice! But as a part of [this](https://github.com/grafana/grafana/issues/33809), we'll have to refactor this and use a rest api endpoint in the plugin backend instead of using the plugin proxy. I guess this will be done for 8.1, so probably not worth putting any more effort into the pluginproxy at this point.

Reply:
Configuring Log Analytics to use different credentials complicates all this (even if we are deprecating it). There's three places we hit ARG for three different reasons. It's a messy venn diagram

 - Requesting client-side data for the resource picker UI for Log Analytics (with LA credentials, if configured)
 - Requesting resource picker UI for Metrics with the Metrics credentials
 - ARG datasource backend queries in PR #33293, with Metrics credentials.

That's why this route has its `resourcepickerdata` name to distinguish it from the endpoint that'll use Metrics credentials. 

This does need tidying up though!!!